requirements.txt

```diff
@@ -1,3 +1,3 @@
 errbot
 github3.py
-IGitt==0.3.0.dev20170612081715
+IGitt==0.3.0.dev20170615131838
```

Comment:
this is a related change, the IGitt new functionality that are used in this commit are in this version